lib/graphiti/sideload.rb

```diff
@@ -70,10 +75,25 @@ def self.link(&blk)
       self.link_proc = blk
     end
 
+    # Todo ApplicationResource
+    def create_remote_resource
```

Comment:
Should we raise an error or maybe a warning if:

```
class PostResource
  self.remote = 'http://my-api.com/api/v1/post'
end

class UserResource
  has_many :posts, remote: 'http://my-api.com/api/v1/post'
end
```

As the user probably meant to use the existing Post resource.  Also we shouldn't allow a user to specify both `remote` and `resource` to the same sideload.

Reply:
I added a check that we can't pass both `resource` and `remote` 👍 

That said, I think the code snippet you posted is WAD and we should solve that with documentation. If someone is surprised by this behavior, it means they don't understand `remote` to begin with.